requirements.txt

```diff
@@ -3,5 +3,5 @@ numpy
 easydict
 opencv-python
 tensorflow
-nltk # required by tensorlayer
-# pycocotools  # must be installed after cython and numpy are installed
+tensorlayer
+pycocotools  # must be installed after cython and numpy are installed
```

Comment:
pycocotools must be installed after cython and numpy are installed, as commented.

Reply:
does pip install follow the order in the requirements.txt?